backend/infrahub/git/tasks.py

```diff
@@ -161,3 +162,82 @@ async def generate_artifact(model: RequestArtifactGenerate) -> None:
         log.exception("Failed to generate artifact", error=exc)
         artifact.status.value = "Error"
         await artifact.save()
+
+
+@flow(name="artifact-definition-generate")
+async def generate_request_artifact_definition(model: RequestArtifactDefinitionGenerate) -> None:
+    log.info(
```

Comment:
we can remove this log, prefect will automatically track the execution of the flow